### PR TITLE
Snackbar Component

### PR DESCRIPTION
**Pull request by sahiba0915**
Files to check:

1. snackbar.html
2. snackbar.css

live preview: https://snackbarcomp.netlify.app/component/snackbar/snackbar.html

### CODE REVIEW

File: component/snackbar/snackbar.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+                
+                <!-- JavaScript -->
+                <script>
+                    function myFunction() {
+                      const x = document.getElementById("snackbar");
+                      x.className = "show";
+                      setTimeout(function () {
+                        x.className = x.className.replace("show", "");
+                      }, 3000);
+                    }
```
Comment (Mohasin-Haque) [inline component/snackbar/snackbar.html:89]: How about adding it to separate js file. But its ok for now to do this.

File: component/snackbar/snackbar.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+.snackbar-description {
+  margin-bottom: 1.5rem;
+  font-size: 1.2rem;
+  color: var(--secondary-text);
+}
+
+#snackbar {
+  visibility: hidden;
+  min-width: 250px;
```
Comment (Mohasin-Haque) [inline component/snackbar/snackbar.css:39]: Why not rem?

**Review by Mohasin-Haque — COMMENTED** [review]
I like the way you did the commit messages and done it in small small commits. You are good to go.

File: component/navigation/navigation.html [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+              />
+              <label for="search-box" class="fas fa-search"></label>
+            </form>
+
+            <div class="icons">
+              <div id="search-btn" class="fas fa-search"></div>
+              <a href="#" class="fas fa-user"></a>
+              <a href="#" class="fa-solid fa-shirt"></a>
+              <a href="#" class="fas fa-heart"></a>
+              <a href="#" class="fas fa-shopping-cart"></a>
```
Comment (irajbhinde) [inline component/navigation/navigation.html:81]: Following element is breaking out of line in preview, check it out once

**Review by irajbhinde — COMMENTED** [review]
Create an example box in future for better visibility of component examples

Looks good to merge otherwise

File: component/snackbar/snackbar.html (hunk 1 above)
Comment (sahiba0915) [reply component/snackbar/snackbar.html:89]: will work on this, thanks

File: component/snackbar/snackbar.css (hunk 2 above)
Comment (sahiba0915) [reply component/snackbar/snackbar.css:39]: ok will use rems

File: component/navigation/navigation.html (hunk 3 above)
Comment (sahiba0915) [reply component/navigation/navigation.html:81]: ok

**Pull request merged**